scripts/release-post/downloadsTable.mjs

```diff
@@ -193,6 +201,14 @@ const resolveDownloads = version => {
     );
   }
 
+  if (semVer.satisfies(version, '< 19.9.0')) {
```

Comment:
I don't think this is needed as new-release blog posts only cover newer versions, and it's rare for someone to republish an older blog post...